src/bin/wallet.rs

```diff
@@ -155,7 +155,7 @@ fn parse_args() -> Result<WalletConfig, Box<error::Error>> {
         path.to_str().unwrap()
     };
     let id = read_keypair(id_path).or_else(|err| {
-        display_actions();
+        println!("{}", matches.usage());
```

Comment:
Should these all be `eprintln!`?

Reply:
I don't have much of an opinion right now, we aren't super consistent about this.  The previous help went to stdout and this patch preserves that behavior.  Happy to add an `e`, lmk

Reply:
Yeah, I think traditionally you'd only write `usage()` to stdout if `--help`.